code/modules/antagonists/vampire/frenzy_vampire.dm

```diff
@@ -104,7 +107,7 @@
 
 /datum/movespeed_modifier/frenzy_speed
 	blacklisted_movetypes = FLYING | FLOATING
-	multiplicative_slowdown = -0.5
+	multiplicative_slowdown = -0.1 // Might seem very low but at this point we are already slow as balls from hunger
```

Comment:
This comment makes no sense, if we are slow as balls from hunger then we would want a larger speedup, but you are reducing the speedup and making them slower.